package.json

```
@@ -7,6 +7,10 @@
   ],
   "scripts": {
     "foreach": "yarn workspaces foreach --no-private --parallel --verbose",
+    "build:tsc": "tsc --build --verbose",
+    "build": "yarn run build:tsc",
+    "build:watch": "concurrently --raw --kill-others 'tsc --build --verbose --watch --inlineSourceMap'",
+    "build:clean": "tsc --build --clean && yarn workspaces foreach exec rm -rf dist",
```

The TypeScript build does not output to the individual `dist` directories so we shouldn't clean them

```suggestion
    "build:clean": "tsc --build --clean",
```